cg/models/analysis.py

```diff
@@ -9,6 +9,9 @@
 class AnalysisModel(BaseModel):
     """Metadata analysis model"""
 
+    class Config:
+        arbitrary_types_allowed = True
+
```

Comment:
Why is this added? Is it needed?

Reply:
Yes, due to the types in `class NextflowAnalysis(AnalysisModel):`